cg/services/run_devices/pacbio/metrics_parser/utils.py

```diff
@@ -32,12 +35,39 @@ def _parse_report_to_model(report_file: Path, data_model: Type[BaseMetrics]) ->
     parsed_json: dict = ReadFile.read_file[FileFormat.JSON](report_file)
     if data_model == SmrtlinkDatasetsMetrics:
         return data_model.model_validate(parsed_json[0], from_attributes=True)
-    metrics: list[dict[str, Any]] = parsed_json.get("attributes")
-    data: dict = {report_field["id"]: report_field["value"] for report_field in metrics}
+    metrics: list[dict[str, Any]] = parsed_json.get(MetricsFileFields.ATTRIBUTES)
+    data: dict = {
+        report_field["id"]: report_field[MetricsFileFields.VALUE_ONE] for report_field in metrics
```

Comment:
Why do we always use the key "id" here? And if we have created the MetricsFileFields class, should that string not be in there?